src/colvar_rotation_derivative.h

```diff
@@ -55,34 +64,53 @@ inline constexpr bool operator&(rotation_derivative_dldq Lhs, rotation_derivativ
 }
 
 /// \brief Helper class for calculating the derivative of rotation
-template <typename T1, typename T2>
+template <typename T1, typename T2, bool soa = false>
```

Comment:
The soa template parameter is now always true. This might enable a slight simplification of the structure and its instantiations.

Reply:
Done in https://github.com/Colvars/colvars/pull/788/commits/c25b79a76538a085fe90cb04a25ec79d8f7ec363. I had to revise the `debug_gradients` function template too, and now it has been moved back to `colvartypes.*`.